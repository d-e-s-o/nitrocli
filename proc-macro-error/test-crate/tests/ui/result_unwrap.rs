extern crate test_crate;
use test_crate::make_fn;

make_fn!(result_unwrap);

fn main() {}
