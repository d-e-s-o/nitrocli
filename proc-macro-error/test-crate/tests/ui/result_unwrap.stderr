error: Result::unwrap_or_abort() test
 --> $DIR/result_unwrap.rs:4:10
  |
4 | make_fn!(result_unwrap);
  |          ^^^^^^^^^^^^^
